apps/ecommerce/frontend/src/components/FieldFallback.tsx

```diff
@@ -0,0 +1,28 @@
+import { Box, Button, Note } from '@contentful/f36-components';
+import { Field as DefaultField } from '@contentful/default-field-editors';
+import { useAutoResizer, useSDK } from '@contentful/react-apps-toolkit';
+import { FieldAppSDK } from '@contentful/app-sdk';
+import { ErrorComponentProps } from '../types';
+import tokens from '@contentful/f36-tokens';
+
+const FieldFallback = ({ error, resetErrorHandler }: ErrorComponentProps) => {
+  const sdk = useSDK<FieldAppSDK>();
+  useAutoResizer();
```

Comment:
This is a fix for if the field loads and instantly errors. This will make sure the field fully resizes to the fallback's height